curriculum/challenges/english/25-front-end-development/review-css/671a9a0a140c2b9d6a75629f.md

```diff
@@ -33,9 +33,15 @@ Review the concepts below to prepare for the upcoming exam.
 
 ## Different Types of CSS Combinators
 
-- **Descendant Combinator**: Selects elements that are descendants of a specified parent. For example, this targets all `li` elements inside `ul` elements.
-- **Child Combinator (`>`)**: Selects elements that are **direct children** of a specified parent. For example, this targets all `p` elements that are direct children of an element with the `container` class.
-- **Next-sibling Combinator (`+`)**: Selects an element that **immediately follows** a specified sibling element. For example, it selects the `p` element that comes right after an `h2`.
+- **Descendant Combinator**: Selects elements that are nested within a specified parent.  
+  For example, `ul li` selects all `li` elements that are descendants of any `ul` element.
```

Comment:
now you have added the examples, which is not what we want to do in the reviews, please review the issue